src/Sonarr.Api.V3/Blocklist/BlocklistController.cs

```diff
@@ -23,12 +25,17 @@ public class BlocklistController : Controller
 
         [HttpGet]
         [Produces("application/json")]
-        public PagingResource<BlocklistResource> GetBlocklist([FromQuery] PagingRequestResource paging)
+        public PagingResource<BlocklistResource> GetBlocklist([FromQuery] PagingRequestResource paging, [FromQuery] int[] seriesIds = null, [FromQuery] DownloadProtocol[] protocols = null)
         {
             var pagingResource = new PagingResource<BlocklistResource>(paging);
             var pagingSpec = pagingResource.MapToPagingSpec<BlocklistResource, NzbDrone.Core.Blocklisting.Blocklist>("date", SortDirection.Descending);
 
-            return pagingSpec.ApplyToPage(_blocklistService.Paged, model => BlocklistResourceMapper.MapToResource(model, _formatCalculator));
+            if (seriesIds != null && seriesIds.Any())
```

Comment:
Not sure if `seriesIds?.Any() == true` is better, but thoughts?

Reply:
switched over to this. looks cleaner for sure.